lib/button/resources/offers.rb

```diff
@@ -0,0 +1,20 @@
+require 'button/resources/resource'
+
+module Button
+  # https://www.usebutton.com/developers/api-reference/
+  #
+  class Offers < Resource
+    def path
+      '/v1/offers'
+    end
+
+    # Retrieve offers that are available to Publishers user
+    #
+    # @param [Hash] user to retrieve offers for
+    # @return [Button::Response] the API response
+    #
+    def get_offers(user)
```

Comment:
nit: `get_offers` seems a bit redundant when you pair it with the namespace.  A few suggestions if you're open to it:

```ruby
client.offers.get(user_id: '123') # ?
client.offers.get_by_user(user_id: '123') # ?
```

Reply:
I was going to do `get_offers`, but this is a POST request, interestingly enough (not sure why we chose POST over GET), so it seemed strange to make it `get_offers`.

I'm ok changing it if you feel strongly about it.